src/frontends/tensorflow_lite/src/graph_iterator_flatbuffer.hpp

```diff
@@ -72,7 +72,7 @@ class GraphIteratorFlatBuffer {
 #else
             std::ifstream tflite_stream(path, std::ios::in | std::ifstream::binary);
 #endif
-            char buf[offset_size * 2] = {};
+            char buf[offset_size * 2 + 1] = {};  // +1 is used to overcome gcc's -Wstringop-overread warning
```

Comment:
Not sure in this fix. May be it is better to cast `offset_size` to `size_t` for this?

Reply:
how will it help?

Reply:
discussed offline, now it is clear